tests/core/Jalasoft.TeamUp.Resumes.Core.Tests/PostResumesCoreTests.cs

```diff
@@ -61,5 +60,47 @@ public void PostResumes_Returns_OkObjectResult()
             var result = this.resumeService.PostResumes(stubResume);
             Assert.IsType<Resume>(result);
         }
+
+        [Fact]
+        public void PostResume_Returns_ValidationException()
+        {
+            var stubResume = new Resume()
+            {
+                Id = Guid.Parse("5a7939fd-59de-44bd-a092-f5d8434584de"),
+                Title = "Name Example",
+                PersonalInformation = new Person()
+                {
+                    FirstName = "Juan Jose",
+                    LastName = "Jimenez Javier",
+                    Birthdate = new DateTime(1995, 2, 2),
+                    Picture = "s.jpg"
+                },
+                Contact = new Contact()
+                {
+                    Email = "Jose Ecos",
+                    Direction = "Calle Bolivar y Presidente Montes",
+                    Phone = 75422166
+                },
+                Summary = "Juan Jose Sumary",
+                Skills = new Skill[]
+                    {
+                        new Skill
+                        {
+                            Id = "184bf2b8-abc1-47da-b383-d0e05ca57d4d",
+                            Name = "C#"
+                        },
+                        new Skill
+                        {
+                            Id = "0947a444-09c6-4281-894a-5e7a4acc38eb",
+                            Name = "API"
+                        }
+                    },
+                CreationDate = DateTime.Now.AddDays(-10),
+                LastUpdate = DateTime.Now
+            };
+            this.mockRepository.Setup(repository => repository.Add(stubResume)).Throws(new FluentValidation.ValidationException("BadRequest"));
```

Comment:
the respository should not be throwing a validation exception, only the service, please fix this and make sure the exception is throw in the service method

Reply:
solved